openwisp_radius/base/models.py

```diff
@@ -152,6 +152,9 @@
     'Whether this field should be disabled, allowed or mandatory '
     'in the user registration API.'
 )
+_GET_REGISTRATION_ENDPOINT_HELP_TEXT = _(
```

Comment:
I didn't notice these messages being prefixed with _GET before, it's a bit confusing to me since _get is usually used as a prefix for functions, not constants.

Anyway, let's just change this one: `_REGISTRATION_ENABLED_HELP_TEXT`.